x-pack/plugins/apm/public/context/UrlParamsContext/resolveUrlParams.ts

```diff
@@ -85,15 +76,6 @@ export function resolveUrlParams(location: Location, state: TimeUrlParams) {
     transactionType,
     searchTerm: toString(searchTerm),
 
-    // path params
-    processorEvent,
-    serviceName,
-    traceIdLink,
-    errorGroupId,
-    serviceNodeName: serviceNodeName
-      ? decodeURIComponent(serviceNodeName)
-      : serviceNodeName,
-
     // ui filters
     environment,
     ...localUIFilters,
```

Comment:
I wish we could get rid of the entire file. Having query params defined globally is so bad (they should be defined with the route).

Reply:
Yes but in another PR.